src/components/TaskEditor.vue

```diff
@@ -188,6 +188,7 @@ export default defineComponent({
     const htmlNotes = ref<string>();
     const customFieldSelectedGids = ref<(string | undefined)[]>([]);
     const projectId = computed(() => asanaStore.selectedProject);
+    const project = computed(() => asanaStore.projects.find((proj) => proj.gid === asanaStore.selectedProject))
```

Comment:
I'm not sure we want to allow an undefined project. I don't think it would yield much good

Reply:
I agree and it should never be undefined. Although, since the find can return undefined, TypeScript complains about this. Any way to get around this?

Reply:
I think you can put in a check that if there is no matching project to just throw an error. Then you either get an error or a project (not a project or undefined)